dodiscover/context.py

```diff
@@ -9,6 +9,9 @@
 class Context:
     """Context of assumptions, domain knowledge and data.
 
+    This should NOT be instantiated directly. One should instead
```

Comment:
To that end, should we continue exporting context in dodiscover/__init__.py, or should we only expose the `make_context` method?

Reply:
Hmm good point. I'll remove from top level import and document that.